### ISSUE TITLE
Required jsonnet version is buggy

### ISSUE THREAD

**Issue by vonHartz** [openai/mujoco-worldgen#9]
As discussed [here](https://github.com/openai/multi-agent-emergence-environments/issues/21#issue-599672057), the jsonnet version listed in requirements.txt leads to issues. I'd propose to change the requirement. Version 0.15 seems to work for me.

**Comment by Jiang1206** [comment]
Hello, did you solve this problem？
`jsonnet-0.15.0` seems to have the same error in https://github.com/openai/multi-agent-emergence-environments/issues/21

**Comment by vonHartz** [comment]
I double checked and jsonnet 0.15 works for me in the example I discussed in https://github.com/openai/multi-agent-emergence-environments/issues/21 .
Do you get exactly the same error with the updated jsonnet?

**Comment by Jiang1206** [comment]
Yes, the problem still exists after updating jsonnet, I am still trying to solve it
